homeassistant/components/template/alarm_control_panel.py

```diff
@@ -191,12 +185,12 @@ def supported_features(self) -> int:
     @property
     def code_format(self):
         """Return one or more digits/characters."""
-        return FORMAT_NUMBER
+        return None
 
     @property
     def code_arm_required(self):
```

Comment:
This property is part of the base alarm panel entity interface.

How is it not represented by this alarm panel?

Reply:
You are right, this method can be entirely removed since the base alarm control panel entity class returns None.

Reply:
After looking at this one more time, it might be possible to still use the code in the arm/disarm type actions.  This isn't very intuitive and might just be that the documentation needs to be updated.  I'm going to close this until I can confirm/deny it.